src/mailer/templates/account-closure-notification.ts

```diff
@@ -0,0 +1,67 @@
+import { Dinero } from 'dinero.js';
+import MailContent from './mail-content';
+import { signatureDutch, signatureEnglish } from './signature';
+import MailTemplate, { Language, MailLanguageMap } from './mail-template';
+
+interface AccountClosureNotificationOptions {
+  name: string;
+  balance: Dinero;
+}
+
+const accountClosureNotificationDutch = new MailContent<AccountClosureNotificationOptions>({
+  getHTML: (context) => `
+<p>Beste ${context.name},</p>
+
+<p>Wij willen u informeren dat uw account bij SudoSOS is gesloten. Op het moment van sluiting was het saldo van uw account:<br>
+<span style="color: red; font-weight: bold; font-size: 20px">${context.balance.toFormat()}</span>.</p>
```

Comment:
Ik zou alleen negatieve saldo's rood maken en positieve saldo's gewoon zwart houden. Rode cijfers betekent vaak negatief en mensen gaan in de war raken.